README.md

```diff
@@ -135,6 +182,26 @@ Show the contents of the blocked ips list:
 bundle exec catadog | jq 'select(.kind=="rc") | .response.body.target_files | .[] | select(.path | test("ASM_DATA/blocked_ips"))'
 ```
 
+## Docker Compose
```

Comment:
Is the Docker Compose section relevant? Could it benefit from additional information?